bidiServer/server.js

```diff
@@ -265,3 +281,24 @@ async function processSessionStatus(params, session, response) {
   }
   return response;
 }
+
+// Events handlers
+
+function addConnectionEventHandlers(browser, connection) {
+  browser.on('disconnected', async () => {
+    respondWithError(connection, {}, "-100", "browser closed");
```

Comment:
```suggestion
    respondWithError(connection, {}, "unknown error", "browser closed");
```

Just a thought. This case isn't in the spec, but the error code is from https://w3c.github.io/webdriver/#dfn-unknown-error

Reply:
done